docs/source/merlin_specification.rst

```diff
@@ -227,9 +227,9 @@ see :doc:`./merlin_variables`.
        label   : ITER.%%
 
   ####################################
-  # Merlin Block (Required)
+  # Merlin Block (Optional)
```

Comment:
The merlin block is still required, just the user part is optional. Sorry about the confusion.

Reply:
Sorry when I made the changes for the documentation I branched off a WIP pull request #376. I have rebased this PR to only include changes related to the updated documentation. Please see #376 for the most recent changes.